src/stateFromElement.js

```diff
@@ -45,9 +46,12 @@ type ParsedBlock = {
 };
 
 type ElementStyles = {[tagName: string]: Style};
+type CustomStyleMap = {[styleName: string]: { [key: string]: string }};
+type CustomCssMapToStyle = {[css: string]: string};
```

Comment:
I'm a little confused why we need this mapping. It seem to map the reverse of CustomStyleMap but I'm unclear on why. Maybe you can explain a little..